cli/internal/app/install_claude_desktop.go

```diff
@@ -0,0 +1,92 @@
+package app
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+
+	"github.com/speakeasy-api/gram/cli/internal/app/logging"
+	"github.com/speakeasy-api/gram/cli/internal/mcp"
+	"github.com/urfave/cli/v2"
+)
+
+func newInstallClaudeDesktopCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "claude-desktop",
+		Usage: "Install a Gram toolset as an MCP server in Claude Desktop",
+		Flags: append(installFlags, &cli.StringFlag{
+			Name:  "output-dir",
+			Usage: "Directory to save the .dxt file (defaults to Downloads folder)",
+		}),
+		Action: doInstallClaudeDesktop,
+	}
+}
+
+func getDownloadsDir() string {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "."
+	}
+	return filepath.Join(homeDir, "Downloads")
+}
```

Comment:
This will work in macOS and windows (maybe) but not Linux. You might want to add a check that the downloads directory exists and fallback to `os.MkdirTemp(...)` or `.`